crates/editor/src/editor.rs

```diff
@@ -8885,7 +8885,7 @@ impl Editor {
         self.style = Some(style);
     }
 
-    #[cfg(any(test, feature = "test-support"))]
+    // #[cfg(any(test, feature = "test-support"))]
```

Comment:
On the assistant2 branch, someone was using this directly. I had to take out this being part of `test-support` to get clippy to pass. Works in the example, won't work in production.